content/blog/bdd-case-study.md

```diff
@@ -0,0 +1,52 @@
++++
+title = "Case-Study Behaviour Driven Development"
+slug = "bdd-case-study"
+date = 2022-07-18T02:32:46Z
+image = ""
+draft = false
+authors = ["Incubyte"]
+description = ""
+tags = ["BDD","TDD", "Software Craftsmanship"]
+categories = ["Playbook", "Software Craftsmanship"]
+type = ""
++++
+
+## Implementing Behavior Driven Deployment methodologies to get better Software out the door - a case study
```

Comment:
Development, not Deployment. Also, check capitalization - don't think Software should be capitalized